docs/manuals/core/entities/scenario-cycle-mgt.md

```diff
@@ -135,6 +135,19 @@ You can define a list of authorized tags in the scenario configuration by specif
 From the scenarios that are created from that configuration, if you add a tag that is not authorized, an exception will
 be raised.
 
+# Export a scenario
```

Comment:
Just make sure section titles are homogeneous.
I can see 'Cycle attributes' below.
This then should be 'Scenario exports'.

Often something like 'Exporting scenarios' is used...